tests/testthat/test-driver-snowflake.R

```diff
@@ -147,17 +147,3 @@ test_that("Workbench-managed credentials are ignored for other accounts", {
     error = TRUE
   )
 })
-
-test_that("snowflake() works against a real account", {
```

Comment:
User/pass login isn't enabled for the new instance, so a test like this one will no longer pass.